src/legacy/core_plugins/navigation/public/index.ts

```diff
@@ -21,12 +21,3 @@
 // Once the new platform is ready, they can get removed
```

Comment:
Now that this file is empty, it can probably be deleted.

Reply:
Removing this one gave me trouble with plugin initialization.
I'll remove it once the SCSS moves to NP.